server/routes/ticket.py

```
@@ -302,5 +300,333 @@ def post(self):
+                "resolved": resolved_count,
+                "closed": closed_count,
+            }
+
+            return restful_response(
+                status="success",
+                message="Ticket stats fetched successfully",
+                data=stats,
+                status_code=200,
+            )
+
+        except Exception as e:
```

Flake8 (7.3.0)

[error] local variable 'e' is assigned to but never used

(F841)